### PR TITLE
zh, en: update command of tidb-operator upgrade

### LINKED ISSUE #1258: Upgrade command of TiDB Operator is missing the namespace parameter
## Error Report

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. What is the URL/path of the document related to this issue?
[deploy-tidb-operator.md](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/deploy-tidb-operator.md)

2. How would you like to improve it?
> 4. 升级 TiDB Operator
    如果需要升级 TiDB Operator，请先修改 ${HOME}/tidb-operator/values-tidb-operator.yaml 文件，然后执行下面的命令进行升级：
    helm upgrade tidb-operator pingcap/tidb-operator -f  ${HOME}/tidb-operator/values-tidb-operator.yaml

The command is is missing the `namespace` parameter, it should be 
```shell
helm upgrade tidb-operator pingcap/tidb-operator --namespace=tidb-admin -f ${HOME}/tidb-operator/values-tidb-operator.yaml
```

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)
Closes #1258 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/deploy-tidb-operator.md [hunk 1]
```diff
@@ -249,5 +249,5 @@ tidbmonitors.pingcap.com             2020-06-11T07:59:41Z
     {{< copyable "shell-regular" >}}
 
     ```shell
-    helm upgrade tidb-operator ./tidb-operator
+    helm upgrade tidb-operator ./tidb-operator --namespace=tidb-admin
```
Comment (csuzhangxc) [inline zh/deploy-tidb-operator.md:252]: no `Customize TiDB Operator` in ZH doc, 😂

**Review by csuzhangxc — APPROVED** [review]
LGTM

File: zh/deploy-tidb-operator.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ tidbmonitors.pingcap.com             2020-06-11T07:59:41Z
     {{< copyable "shell-regular" >}}
 
     ```shell
-    helm upgrade tidb-operator pingcap/tidb-operator -f  ${HOME}/tidb-operator/values-tidb-operator.yaml
+    helm upgrade tidb-operator pingcap/tidb-operator --namespace=tidb-admin -f  ${HOME}/tidb-operator/values-tidb-operator.yaml
```
Comment (qiancai) [inline zh/deploy-tidb-operator.md:136]: 英文版本里，${HOME}/tidb-operator/values-tidb-operator.yaml 的外面包裹了 `` 中文需要吗

Comment (KanShiori) [reply zh/deploy-tidb-operator.md:136]: 应该是都不需要的，我修改下

File: zh/deploy-tidb-operator.md (hunk 1 above)
Comment (qiancai) [reply zh/deploy-tidb-operator.md:252]: The Chinese version of the **Customize TiDB Operator** section will be added by https://github.com/pingcap/docs-tidb-operator/pull/1266.

**Review by qiancai — APPROVED** [review]

**Pull request merged**